float x
float y
int8 needed_uav
int8 needed_ugv
---
bool done